setup.cfg

```diff
@@ -93,7 +93,7 @@ install_requires =
     deprecated>=1.2.13
     dill>=0.2.2, <0.4
     flask>=1.1.0, <2.0
-    flask-appbuilder~=3.4, <4.0.0
+    flask-appbuilder==3.4.4
```

Comment:
```suggestion
    # We are tightly coupled with FAB version because we vendored in part of FAB code related to security manager
    # This is done as part of preparation to removing FAB as dependency, but we are not ready for it yet
    # Every we update FAB version here, please make sure that you review TODO @jhtimmins - please describe 
    # what to compare and how to sync the changes :)
    flask-appbuilder==3.4.4
```